bindings/go/blst_headers/blst.h

```diff
@@ -0,0 +1,483 @@
+/*
```

Comment:
Is this file generated?

Reply:
Unfortunately no, it was manually copied. It will need to be updated manually if it changes.

Reply:
Maybe we could remove the mentions of swig to avoid confusion?

Reply:
Maybe. How about a CI check that compares those files to the bindings in blst & fails if they don't perfectly match?